crates/bevy_log/src/lib.rs

```diff
@@ -98,13 +98,22 @@ pub struct LogSettings {
     /// Filters out logs that are "less than" the given level.
     /// This can be further filtered using the `filter` setting.
     pub level: Level,
+
+    /// An identifier, in reverse DNS notation, used to identify the subsystem that is being logged.
+    /// Used for iOS.
+    pub subsystem: String,
```

Comment:
SInce it's only used in iOS should this be conditionally added to the struct?

Reply:
Done, thanks.